### ISSUE TITLE
getFile issue: file not encoded properly?

### ISSUE THREAD

**Issue by avimar** [phaxio/phaxio-node#4]
Sample README code:

```
phaxio.faxes.getFile({ id: 987 })
  .then(fileString => fs.writeFileSync(`${__dirname}/full_file.pdf`, fileString))
  .catch((err) => { throw err; });
```

My code:

```
const Phaxio = require('phaxio-official');
const phaxio = new Phaxio(Common.conf.get('phaxioKey'), Common.conf.get('phaxioSecret'));
const fs = require('fs');

var id = 143236766;
const filename = 'fax-'+id+'.pdf';
const fileData = await phaxio.faxes.getFile({id:id});
await fs.promises.writeFile(filename, fileData);
```
This file seems to not be a valid PDF, and it's double the size of the file downloaded from the website.

Any suggestions on encoding issues?

I tried setting `{encoding: X}` with these options but none helped: ascii. base64, hex, ucs2, utf8, binary.

**Comment by avimar** [comment]
I figured it out, seemingly it never worked. Request was not configured for downloading the binary file. (Also, thumbnail option doesn't seem to work.)

`encoding` - encoding to be used on `setEncoding` of response data. If `null`, the `body` is returned as a `Buffer`. Anything else **(including the default value of `undefined`)** will be passed as the [encoding](http://nodejs.org/api/buffer.html#buffer_buffer) parameter to `toString()` (meaning this is effectively `utf8` by default). (**Note:** if you expect binary data, you should set `encoding: null`.)

Adding an `encoding:null` here seems to fix it: https://github.com/phaxio/phaxio-node/blob/77369458cfa3949b2f97c319ff0ad2e16747dc70/src/faxes/shared-methods.js#L88

**Comment by avimar** [comment]
p.s. check if any other downloads are affected, e.g. `get hosted document`

**Comment by jeremy-j-ackso** [comment]
Thanks for the PR!

I did some other updates and just published v1.2.1

`npm install --save phaxio-official@^1.2.1`

**Comment by avimar** [comment]
Thanks!

I didn't see any updates on hosted documents or thumbnails, which is still broken:
`const fileData = await phaxio.faxes.getFile({id:id,thumbnail: 'l'});`

`UnhandledPromiseRejectionWarning: TypeError: Cannot set property 'thumbnail' of undefined`

**Comment by jeremy-j-ackso** [comment]
@avimar I just published v1.2.2 that addresses that problem. Sorry I didn't get it last time!
